src/adapter/activateDaffodilDebug.ts

```diff
@@ -168,35 +168,52 @@ function createDebugRunFileConfigs(
         tdmlConfig.action = tdmlAction
 
         if (tdmlAction === 'execute') {
-          tdmlConfig.name = '${command:AskForTDMLName}'
-          tdmlConfig.description = '${command:AskForTDMLDescription}'
-          tdmlConfig.path = targetResource.fsPath
+          tdmlConfig.path = targetResource.fsPath.toString()
+
+          tdmlConfig.name =
+            tdmlConfig.name ||
+            (await vscode.commands.executeCommand(
+              'extension.dfdl-debug.getTDMLName',
+              tdmlConfig.path
+            ))
+
+          tdmlConfig.description =
+            tdmlConfig.description ||
+            (await vscode.commands.executeCommand(
+              'extension.dfdl-debug.getTDMLDescription',
+              tdmlConfig.path
+            ))
+
+          if (!tdmlConfig.name || !tdmlConfig.description) {
+            vscode.window.showErrorMessage(
```

Comment:
I'm not sure that we want this error message here. It makes sense to have one in the case where no test cases are found, but it doesn't make sense if the user presses Esc. Maybe we can meet in the middle and do a toast message in the bottom right, the same way we do for the Infoset being created?